platforms/native/language-native/src/integTest/groovy/org/gradle/language/swift/SwiftIncrementalCompileIntegrationTest.groovy

```diff
@@ -245,8 +245,7 @@ class SwiftIncrementalCompileIntegrationTest extends AbstractInstalledToolChainI
         outputs.recompiledClasses('main', 'sum', 'greeter', 'multiply')
     }
 
-    @RequiresInstalledToolChain(ToolChainRequirement.SWIFTC_4)
-    @ToBeFixedForConfigurationCache
+    @RequiresInstalledToolChain(ToolChainRequirement.SWIFTC_5)
```

Comment:
Changed test to use the most recent version of Swift so it is more likely to run.